admin/js/monk-admin.js

```diff
@@ -66,6 +66,38 @@
 			return false;
 		});
 
+		$( document ).on( 'submit', '#monk-tools-form', function( event ) {
+			event.preventDefault();
+			$( '#monk-spinner' ).addClass( 'is-active' );
+			var form_data = $( '#monk-tools-form' ).serializeArray();
+
+			form_data.push({
```

Comment:
Use a hidden field on form and remove this.
`<input type="hidden" name="action" value="monk_set_language_to_elements" />`